src/main/java/com/hobbycollection/HobbyCollectionRESTController.java

```diff
@@ -16,6 +17,7 @@
      * @return URL
      */
     @GetMapping("/api/Collection/fetchByid")
+    @Cacheable(value="collection", key="#id")
```

Comment:
Caching is a good concept, but I wouldn't put it on the controller.  It's best on the service layer.

Also, what about ejecting cache?  I don't see a matching @CacheEject annotation.